src/ProjectOrigin.Vault/Activities/RequestStatusArgs.cs

```diff
@@ -0,0 +1,9 @@
+using System;
+
+namespace ProjectOrigin.Vault.Activities;
+
+public record RequestStatusArgs
```

Comment:
its hard to tell context from this name. It's highly generic.

Status of what? Is it just something generic?